crypto/Pos/Crypto/Signing/Signing.hs

```diff
@@ -180,4 +181,4 @@ proxyVerify t ProxySignature{..} omegaPred m =
                  , signTag t
                  , Bi.serialize' m
                  ])
-            psigSig
+            (psigSig psig)
```

Comment:
This style was introduced by neongreen, nothing i can do.

Reply:
Which style? AFAIU, this code is equivalent to the previous version, you only did some cosmetic changes (which are subjective and I can't say you made this code better). It's better not to do them (or at least do them in a separate commit), because it complicates code review.
If you changed anything else in this module, then reviewing these cosmetic changes would be a bit less annoying.
Maybe that's just a leftover, because you had to revert smth.